pie/models/attention.py

```diff
@@ -163,7 +163,8 @@ def forward(self, dec_out, enc_outs, lengths):
         # (batch x src_seq_len) => (trg_seq_len x batch x src_seq_len)
         mask = mask.unsqueeze(0).expand_as(weights)
         # weights = weights * mask.float()
-        weights.masked_fill_(1 - mask, -float('inf'))
+        # Torch 1.1 -> 1.2: (1 - mask) becomes ~(mask)
+        weights.masked_fill_(~(mask), -float('inf'))
```

Comment:
I think you can just use `~mask~` (no need for parens)

Reply:
I'll see if this works. I'll probably bump the version if you are okay to make a release :) Last torch allows for easier use of current CUDNN :)